examples/stacks/laravel/devbox.d/php/php-fpm.conf

```diff
@@ -0,0 +1,16 @@
+[global]
+pid = ${PHPFPM_PID_FILE}
+error_log = ${PHPFPM_ERROR_LOG_FILE}
+
+[www]
+; user = www-data
+; group = www-data
+listen = 127.0.0.1:${PHPFPM_PORT}
```

Comment:
uh, this might not work for devbox.sh. Maybe try this example on a VM? Usually `0.0.0.0` will do it